pkg/macaroon/macaroon_test.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package macaroon_test
```

Comment:
More test suggestions:
Verify that `AddFirstPartyCaveat` returns an identical macaroon with two separate calls adding the same caveat
Verify that it returns two different macaroons when `AddFirstPartyCaveat` is called on the same macaroon, but with two different caveats
Verify that it returns two different macaroons when `AddFirstPartyCaveat` is called on two different macaroons, but with the same caveat

Reply:
Maybe not all of these cases are necessary, so implement at your own discretion :)